src/status_im/common/enter_seed_phrase/view.cljs

```diff
@@ -229,8 +232,9 @@
   []
   (let [{:keys [on-success]} (rf/sub [:get-screen-params])]
     (rn/use-unmount
-     (when-not on-success
-       #(rf/dispatch [:onboarding/clear-navigated-to-enter-seed-phrase-from-screen])))
+     (fn []
```

Comment:
could please elaborate on why `(when-not on-success` was removed

Reply:
Hi @flexsurfer, Thank you for reviewing the PR.

> could please elaborate on why (when-not on-success was removed

Now, we are also passing on-success for onboarding instead of hardcoding it, which is why I removed it. It's a small, harmless cleanup event, so it's not problematic if it's being called every time. However, I have removed it and am now passing it as an on-unmount parameter. Thank you